PSGitHub.psm1

```diff
@@ -27,5 +27,11 @@ foreach ($File in $FunctionList) {
 }
 #endregion
 
+#region aliases
+
+New-Alias -Name Fork-GitHubGist -Value Copy-GitHubGist
```

Comment:
Let's define aliases in a separate file, and load those. I'd suggest creating an `\Aliases` subdirectory.